packages/components/text-field/src/text-field.ts

```diff
@@ -15,6 +15,7 @@ let nextUniqueId = 0;
  * Single line text field component.
  *
  * @csspart wrapper - The input's wrapper
+ * @cssprop --sl-textarea-rows - The number of rows initially visible in the textarea
```

Comment:
This one should be added to the `textarea.ts` file, right? :)

Reply:
Right :)